manifests/bind_event.pp

```diff
@@ -0,0 +1,34 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue in katello with the correct messages from candlepin to filter out uneeded messages
+#
+# === Parameters
+#
+# $hostname::                   FQDN of the host
+#
+# $ss_cert::                    SSL cert to use for qpid-config commands
+#
+# $candlepin_event_queue::      Name of the Candlepin event queue
+#
+# $title::                      Title of event
+define qpid::bind_event(
+include qpid::params
+  String $hostname = $::fqdn,
+  String $ssl_cert = '/etc/pki/katello/qpid_client_striped.crt',
```

Comment:
This module isn't katello specific, there are non-katello users, leave this undef and pass it in, you should also try to support not having one, i.e. connecting over non-ssl